run_unittests.py

```diff
@@ -2293,6 +2293,10 @@ def test_configure_file_warnings(self):
         self.assertRegex(out, "WARNING:.*'FOO_BAR'.*nosubst-nocopy2.txt.in.*not present.*")
         self.assertRegex(out, "WARNING:.*'empty'.*config.h.in.*not present.*")
         self.assertRegex(out, "WARNING:.*empty configuration_data.*test.py.in")
+        # Warnings for configuration files that are overwritten.
+        self.assertRegex(out, "WARNING:.*double_output.txt.*overwrites")
+        self.assertRegex(out, "WARNING:.*subdir/double_output2.txt.*overwrites")
```

Comment:
Test failure on Windows is because of the `/` here. You could replace it with `.` because it's a regex.